.circleci/config.yml

```diff
@@ -5,37 +5,37 @@ version: 2.1
 executors:
     py3:
         docker:
-            - image: pyne/ubuntu_18.04_py3_pyne-deps:latest
+            - image: ghcr.io/pyne/ubuntu_18.04_py3_pyne-deps:latest
               auth:
```

Comment:
Do we still need the authorization once we move to GHCR?

Reply:
no we don't